app/controllers/application_controller.rb

```diff
@@ -1307,7 +1411,8 @@ def show_index_of_objects(query, args = {})
                                          name: name.display_name)
         when :pattern_search
           :runtime_no_matches_pattern.t(type: type,
-                                        value: query.params[:pattern].to_s).html_safe
+                                        value: query.params[:pattern].
+                                                     to_s).html_safe
```

Comment:
Reasonable point.  However, it's just an error message so it would be worth removing the html_safe and seeing if any tests fail or any weird things happen when you run the code.  If not, then we should remove it.  If it does cause a failure of some sort, then we'll need to see if there's some way this could turn into an HTML injection issue which seems like the only real risk here.